{"severity":"error","source":"zedagent","iid":"2151","content":"subWwanMetrics.Get failed: Get(nim/WwanMetrics) unknown key global","msgid":"2619","timestamp":"2023-09-27T07:02:19.116234390Z","filename":"/pillar/cmd/zedagent/handlemetrics.go:877","function":"github.com/lf-edge/eve/pkg/pillar/cmd/zedagent.getCellularMetrics"}{"severity":"error","source":"zedagent","iid":"2151","content":"subWwanMetrics.Get failed: Get(nim/WwanMetrics) unknown key global","msgid":"2620","timestamp":"2023-09-27T07:03:03.303578805Z","filename":"/pillar/cmd/zedagent/handlemetrics.go:877","function":"github.com/lf-edge/eve/pkg/pillar/cmd/zedagent.getCellularMetrics"}{"severity":"error","source":"zedagent","iid":"2151","content":"subWwanMetrics.Get failed: Get(nim/WwanMetrics) unknown key global","msgid":"2621","timestamp":"2023-09-27T07:03:29.526610689Z","filename":"/pillar/cmd/zedagent/handlemetrics.go:877","function":"github.com/lf-edge/eve/pkg/pillar/cmd/zedagent.getCellularMetrics"}{"severity":"error","source":"zedagent","iid":"2151","content":"subWwanMetrics.Get failed: Get(nim/WwanMetrics) unknown key global","msgid":"2622","timestamp":"2023-09-27T07:04:01.759648492Z","filename":"/pillar/cmd/zedagent/handlemetrics.go:877","function":"github.com/lf-edge/eve/pkg/pillar/cmd/zedagent.getCellularMetrics"}{"severity":"error","source":"zedagent","iid":"2151","content":"subWwanMetrics.Get failed: Get(nim/WwanMetrics) unknown key global","msgid":"2623","timestamp":"2023-09-27T07:04:53.554428383Z","filename":"/pillar/cmd/zedagent/handlemetrics.go:877","function":"github.com/lf-edge/eve/pkg/pillar/cmd/zedagent.getCellularMetrics"}{"severity":"error","source":"zedagent","iid":"2151","content":"subWwanMetrics.Get failed: Get(nim/WwanMetrics) unknown key global","msgid":"2624","timestamp":"2023-09-27T07:05:35.551307088Z","filename":"/pillar/cmd/zedagent/handlemetrics.go:877","function":"github.com/lf-edge/eve/pkg/pillar/cmd/zedagent.getCellularMetrics"}{"severity":"error","source":"zedagent","iid":"2151","content":"subWwanMetrics.Get failed: Get(nim/WwanMetrics) unknown key global","msgid":"2625","timestamp":"2023-09-27T07:06:29.594792935Z","filename":"/pillar/cmd/zedagent/handlemetrics.go:877","function":"github.com/lf-edge/eve/pkg/pillar/cmd/zedagent.getCellularMetrics"}{"severity":"error","source":"zedrouter","iid":"2151","content":"client.Do fail: ECONNREFUSED","msgid":"2626","timestamp":"2023-09-27T07:06:45.934602916Z","filename":"/pillar/zedcloud/send.go:974","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnIntf"}{"severity":"error","source":"zedrouter","iid":"2151","content":"All attempts to connect to http://192.168.10.129:9095 failed: send via eth0 with src IP 192.168.10.168: Get \"http://192.168.10.129:9095\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2627","timestamp":"2023-09-27T07:06:45.934797877Z","filename":"/pillar/zedcloud/send.go:1127","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnIntf"}{"severity":"error","source":"zedagent","iid":"2151","content":"client.Do fail: ECONNREFUSED","msgid":"2628","timestamp":"2023-09-27T07:06:47.840422907Z","filename":"/pillar/zedcloud/send.go:974","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnIntf"}{"severity":"error","source":"zedagent","iid":"2151","content":"All attempts to connect to https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/config failed: send via eth0 with src IP 192.168.10.168: Post \"https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/config\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2629","timestamp":"2023-09-27T07:06:47.840574676Z","filename":"/pillar/zedcloud/send.go:1127","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnIntf"}{"severity":"error","source":"zedagent","iid":"2151","content":"All attempts to connect to 192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/config failed: send via eth0 with src IP 192.168.10.168: Post \"https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/config\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2630","timestamp":"2023-09-27T07:06:47.840661789Z","filename":"/pillar/zedcloud/send.go:266","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnAllIntf"}{"severity":"info","source":"zedagent","iid":"2151","content":"UpdateLedManagerConfig: set 3 was 4","msgid":"2631","timestamp":"2023-09-27T07:06:47.840831419Z","filename":"/pillar/utils/ledmanagerutils.go:33","function":"github.com/lf-edge/eve/pkg/pillar/utils.UpdateLedManagerConfig"}{"severity":"info","source":"zedagent","iid":"2151","content":"Zedagent status modify","msgid":"2632","timestamp":"2023-09-27T07:06:47.841760954Z","filename":"/pillar/types/zedagenttypes.go:607","function":"github.com/lf-edge/eve/pkg/pillar/types.ZedAgentStatus.LogModify"}{"severity":"info","source":"volumemgr","iid":"2151","content":"Zedagent status modify","msgid":"2633","timestamp":"2023-09-27T07:06:47.844522665Z","filename":"/pillar/types/zedagenttypes.go:607","function":"github.com/lf-edge/eve/pkg/pillar/types.ZedAgentStatus.LogModify"}{"severity":"info","source":"baseosmgr","iid":"2151","content":"Zedagent status modify","msgid":"2634","timestamp":"2023-09-27T07:06:47.854515997Z","filename":"/pillar/types/zedagenttypes.go:607","function":"github.com/lf-edge/eve/pkg/pillar/types.ZedAgentStatus.LogModify"}{"severity":"info","source":"zedmanager","iid":"2151","content":"Zedagent status modify","msgid":"2635","timestamp":"2023-09-27T07:06:47.854951930Z","filename":"/pillar/types/zedagenttypes.go:607","function":"github.com/lf-edge/eve/pkg/pillar/types.ZedAgentStatus.LogModify"}{"severity":"info","source":"nodeagent","iid":"2151","content":"Zedagent status modify","msgid":"2636","timestamp":"2023-09-27T07:06:47.855297146Z","filename":"/pillar/types/zedagenttypes.go:607","function":"github.com/lf-edge/eve/pkg/pillar/types.ZedAgentStatus.LogModify"}{"severity":"info","source":"diag","iid":"2151","content":"Zedagent status modify","msgid":"2637","timestamp":"2023-09-27T07:06:47.855617409Z","filename":"/pillar/types/zedagenttypes.go:607","function":"github.com/lf-edge/eve/pkg/pillar/types.ZedAgentStatus.LogModify"}{"severity":"info","source":"nim","iid":"2151","content":"Zedagent status modify","msgid":"2638","timestamp":"2023-09-27T07:06:47.855966376Z","filename":"/pillar/types/zedagenttypes.go:607","function":"github.com/lf-edge/eve/pkg/pillar/types.ZedAgentStatus.LogModify"}{"severity":"error","source":"zedagent","iid":"2151","content":"client.Do fail: ECONNREFUSED","msgid":"2639","timestamp":"2023-09-27T07:06:47.889426385Z","filename":"/pillar/zedcloud/send.go:974","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnIntf"}{"severity":"error","source":"zedagent","iid":"2151","content":"All attempts to connect to https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/info failed: send via eth0 with src IP 192.168.10.168: Post \"https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/info\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2640","timestamp":"2023-09-27T07:06:47.889566914Z","filename":"/pillar/zedcloud/send.go:1127","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnIntf"}{"severity":"error","source":"zedagent","iid":"2151","content":"All attempts to connect to 192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/info failed: send via eth0 with src IP 192.168.10.168: Post \"https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/info\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2641","timestamp":"2023-09-27T07:06:47.889655604Z","filename":"/pillar/zedcloud/send.go:266","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnAllIntf"}{"severity":"info","source":"zedagent","iid":"2151","content":"handleDeferred() the rest to be sent: 1","msgid":"2642","timestamp":"2023-09-27T07:06:47.889726042Z","filename":"/pillar/zedcloud/deferred.go:227","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.(*DeferredContext).handleDeferred"}{"severity":"info","source":"zedagent","iid":"2151","content":"deferred queue has INFO: 1","msgid":"2643","timestamp":"2023-09-27T07:06:47.889786647Z","filename":"/pillar/cmd/zedagent/zedagent.go:2472","function":"github.com/lf-edge/eve/pkg/pillar/cmd/zedagent.getDeferredSentHandlerFunction.func1"}{"severity":"info","source":"zedagent","iid":"2151","content":"processQueueTask: some deferred items remain to be sent","msgid":"2644","timestamp":"2023-09-27T07:06:47.890196719Z","filename":"/pillar/zedcloud/deferred.go:125","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.(*DeferredContext).processQueueTask"}{"severity":"error","source":"zedagent","iid":"2151","content":"subWwanMetrics.Get failed: Get(nim/WwanMetrics) unknown key global","msgid":"2645","timestamp":"2023-09-27T07:07:01.343185460Z","filename":"/pillar/cmd/zedagent/handlemetrics.go:877","function":"github.com/lf-edge/eve/pkg/pillar/cmd/zedagent.getCellularMetrics"}{"severity":"error","source":"zedagent","iid":"2151","content":"client.Do fail: ECONNREFUSED","msgid":"2646","timestamp":"2023-09-27T07:07:01.378266163Z","filename":"/pillar/zedcloud/send.go:974","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnIntf"}{"severity":"error","source":"zedagent","iid":"2151","content":"All attempts to connect to https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/metrics failed: send via eth0 with src IP 192.168.10.168: Post \"https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/metrics\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2647","timestamp":"2023-09-27T07:07:01.378410743Z","filename":"/pillar/zedcloud/send.go:1127","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnIntf"}{"severity":"error","source":"zedagent","iid":"2151","content":"All attempts to connect to 192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/metrics failed: send via eth0 with src IP 192.168.10.168: Post \"https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/metrics\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2648","timestamp":"2023-09-27T07:07:01.378518675Z","filename":"/pillar/zedcloud/send.go:266","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnAllIntf"}{"severity":"error","source":"zedagent","iid":"2151","content":"sendMetricsProtobufByURL status 1 failed: All attempts to connect to 192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/metrics failed: send via eth0 with src IP 192.168.10.168: Post \"https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/metrics\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2649","timestamp":"2023-09-27T07:07:01.378602201Z","filename":"/pillar/cmd/zedagent/handlemetrics.go:1531","function":"github.com/lf-edge/eve/pkg/pillar/cmd/zedagent.sendMetricsProtobufByURL"}{"severity":"error","source":"nim","iid":"2151","content":"client.Do fail: ECONNREFUSED","msgid":"2650","timestamp":"2023-09-27T07:07:06.441283834Z","filename":"/pillar/zedcloud/send.go:974","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnIntf"}{"severity":"error","source":"nim","iid":"2151","content":"All attempts to connect to https://192.168.10.129:9095/api/v2/edgedevice/ping failed: send via eth0 with src IP 192.168.10.168: Get \"https://192.168.10.129:9095/api/v2/edgedevice/ping\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2651","timestamp":"2023-09-27T07:07:06.441468914Z","filename":"/pillar/zedcloud/send.go:1127","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnIntf"}{"severity":"error","source":"nim","iid":"2151","content":"Zedcloud un-reachable via interface eth0: All attempts to connect to https://192.168.10.129:9095/api/v2/edgedevice/ping failed: send via eth0 with src IP 192.168.10.168: Get \"https://192.168.10.129:9095/api/v2/edgedevice/ping\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2652","timestamp":"2023-09-27T07:07:06.441569661Z","filename":"/pillar/zedcloud/send.go:380","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.VerifyAllIntf"}{"severity":"error","source":"nim","iid":"2151","content":"All attempts to connect to 192.168.10.129:9095/api/v2/edgedevice/ping failed: send via eth0 with src IP 192.168.10.168: Get \"https://192.168.10.129:9095/api/v2/edgedevice/ping\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2653","timestamp":"2023-09-27T07:07:06.441680551Z","filename":"/pillar/zedcloud/send.go:444","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.VerifyAllIntf"}{"severity":"error","source":"nim","iid":"2151","content":"TestConnectivity: Remote temporary failure (endpoint: 192.168.10.129:9095): All attempts to connect to 192.168.10.129:9095/api/v2/edgedevice/ping failed: send via eth0 with src IP 192.168.10.168: Get \"https://192.168.10.129:9095/api/v2/edgedevice/ping\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2654","timestamp":"2023-09-27T07:07:06.441779161Z","filename":"/pillar/conntester/zedcloud.go:135","function":"github.com/lf-edge/eve/pkg/pillar/conntester.(*ZedcloudConnectivityTester).TestConnectivity"}{"severity":"info","source":"nim","iid":"2151","content":"DevicePortConfig port modify","msgid":"2655","timestamp":"2023-09-27T07:07:06.443310826Z","filename":"/pillar/types/zedroutertypes.go:597","function":"github.com/lf-edge/eve/pkg/pillar/types.DevicePortConfig.LogModify"}{"severity":"info","source":"zedagent","iid":"2151","content":"handleDPCLImpl: has real change.","msgid":"2656","timestamp":"2023-09-27T07:07:06.496211265Z","filename":"/pillar/cmd/zedagent/zedagent.go:2200","function":"github.com/lf-edge/eve/pkg/pillar/cmd/zedagent.handleDPCLImpl"}{"severity":"info","source":"nim","iid":"2151","content":"DeviceNetworkStatus port modify","msgid":"2657","timestamp":"2023-09-27T07:07:06.537737899Z","filename":"/pillar/types/zedroutertypes.go:1365","function":"github.com/lf-edge/eve/pkg/pillar/types.DeviceNetworkStatus.LogModify"}{"severity":"warning","source":"nim","iid":"2151","content":"testConnectivityToCloud: remoteTemporaryFailure: Remote temporary failure (endpoint: 192.168.10.129:9095): All attempts to connect to 192.168.10.129:9095/api/v2/edgedevice/ping failed: send via eth0 with src IP 192.168.10.168: Get \"https://192.168.10.129:9095/api/v2/edgedevice/ping\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2658","timestamp":"2023-09-27T07:07:06.537869047Z","filename":"/pillar/dpcmanager/verify.go:442","function":"github.com/lf-edge/eve/pkg/pillar/dpcmanager.(*DpcManager).testConnectivityToCloud"}{"severity":"info","source":"wstunnelclient","iid":"2151","content":"DeviceNetworkStatus port modify","msgid":"2659","timestamp":"2023-09-27T07:07:06.554367795Z","filename":"/pillar/types/zedroutertypes.go:1365","function":"github.com/lf-edge/eve/pkg/pillar/types.DeviceNetworkStatus.LogModify"}{"severity":"error","source":"zedagent","iid":"2151","content":"client.Do fail: ECONNREFUSED","msgid":"2660","timestamp":"2023-09-27T07:07:06.560028372Z","filename":"/pillar/zedcloud/send.go:974","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnIntf"}{"severity":"error","source":"zedagent","iid":"2151","content":"All attempts to connect to https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/info failed: send via eth0 with src IP 192.168.10.168: Post \"https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/info\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2661","timestamp":"2023-09-27T07:07:06.560097591Z","filename":"/pillar/zedcloud/send.go:1127","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnIntf"}{"severity":"error","source":"zedagent","iid":"2151","content":"All attempts to connect to 192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/info failed: send via eth0 with src IP 192.168.10.168: Post \"https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/info\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2662","timestamp":"2023-09-27T07:07:06.560141781Z","filename":"/pillar/zedcloud/send.go:266","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnAllIntf"}{"severity":"info","source":"zedagent","iid":"2151","content":"handleDeferred() the rest to be sent: 1","msgid":"2663","timestamp":"2023-09-27T07:07:06.560165884Z","filename":"/pillar/zedcloud/deferred.go:227","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.(*DeferredContext).handleDeferred"}{"severity":"info","source":"zedagent","iid":"2151","content":"deferred queue has INFO: 1","msgid":"2664","timestamp":"2023-09-27T07:07:06.560186398Z","filename":"/pillar/cmd/zedagent/zedagent.go:2472","function":"github.com/lf-edge/eve/pkg/pillar/cmd/zedagent.getDeferredSentHandlerFunction.func1"}{"severity":"info","source":"zedagent","iid":"2151","content":"processQueueTask: some deferred items remain to be sent","msgid":"2665","timestamp":"2023-09-27T07:07:06.560206372Z","filename":"/pillar/zedcloud/deferred.go:125","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.(*DeferredContext).processQueueTask"}{"severity":"info","source":"downloader","iid":"2151","content":"DeviceNetworkStatus port modify","msgid":"2666","timestamp":"2023-09-27T07:07:06.572931965Z","filename":"/pillar/types/zedroutertypes.go:1365","function":"github.com/lf-edge/eve/pkg/pillar/types.DeviceNetworkStatus.LogModify"}{"severity":"info","source":"zedagent","iid":"2151","content":"DeviceNetworkStatus port modify","msgid":"2667","timestamp":"2023-09-27T07:07:06.575280694Z","filename":"/pillar/types/zedroutertypes.go:1365","function":"github.com/lf-edge/eve/pkg/pillar/types.DeviceNetworkStatus.LogModify"}{"severity":"info","source":"domainmgr","iid":"2151","content":"DeviceNetworkStatus port modify","msgid":"2668","timestamp":"2023-09-27T07:07:06.577710224Z","filename":"/pillar/types/zedroutertypes.go:1365","function":"github.com/lf-edge/eve/pkg/pillar/types.DeviceNetworkStatus.LogModify"}{"severity":"info","source":"ledmanager","iid":"2151","content":"DeviceNetworkStatus port modify","msgid":"2669","timestamp":"2023-09-27T07:07:06.578889292Z","filename":"/pillar/types/zedroutertypes.go:1365","function":"github.com/lf-edge/eve/pkg/pillar/types.DeviceNetworkStatus.LogModify"}{"severity":"info","source":"diag","iid":"2151","content":"DeviceNetworkStatus port modify","msgid":"2670","timestamp":"2023-09-27T07:07:06.579967353Z","filename":"/pillar/types/zedroutertypes.go:1365","function":"github.com/lf-edge/eve/pkg/pillar/types.DeviceNetworkStatus.LogModify"}{"severity":"info","source":"diag","iid":"2151","content":"handleDNSImpl: important change   types.DeviceNetworkStatus{\n  \t... // 4 identical fields\n  \tCurrentIndex: 1,\n  \tRadioSilence: {ChangeRequestedAt: s\"2023-09-27 06:31:33.524749679 +0000 UTC\"},\n  \tPorts: []types.NetworkPortStatus{\n  \t\t{\n  \t\t\t... // 21 identical fields\n  \t\t\tProxyConfig:  {},\n  \t\t\tL2LinkConfig: {},\n  \t\t\tTestResults: types.TestResults{\n- \t\t\t\tLastFailed:    s\"2023-09-27 06:31:39.463394959 +0000 UTC\",\n+ \t\t\t\tLastFailed:    s\"2023-09-27 07:07:06.441656232 +0000 UTC\",\n  \t\t\t\tLastSucceeded: s\"2023-09-27 07:02:06.279727056 +0000 UTC\",\n- \t\t\t\tLastError:     \"\",\n+ \t\t\t\tLastError:     `All attempts to connect to https://192.168.10.129:9095/api/v2/edgedevice/ping failed: send via eth0 with src IP 192.168.10.168: Get \"https://192.168.10.129:9095/api/v2/edgedevice/ping\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection re`...,\n  \t\t\t},\n  \t\t},\n  \t},\n  }\n","msgid":"2671","timestamp":"2023-09-27T07:07:06.581023156Z","filename":"/pillar/cmd/diag/diag.go:615","function":"github.com/lf-edge/eve/pkg/pillar/cmd/diag.handleDNSImpl"}{"severity":"info","source":"zedrouter","iid":"2151","content":"DeviceNetworkStatus port modify","msgid":"2672","timestamp":"2023-09-27T07:07:06.586465576Z","filename":"/pillar/types/zedroutertypes.go:1365","function":"github.com/lf-edge/eve/pkg/pillar/types.DeviceNetworkStatus.LogModify"}{"severity":"info","source":"loguploader","iid":"2151","content":"DeviceNetworkStatus port modify","msgid":"2673","timestamp":"2023-09-27T07:07:06.587007851Z","filename":"/pillar/types/zedroutertypes.go:1365","function":"github.com/lf-edge/eve/pkg/pillar/types.DeviceNetworkStatus.LogModify"}{"severity":"warning","source":"zedagent","iid":"2151","content":"All management ports are marked failed; trying all","msgid":"2674","timestamp":"2023-09-27T07:07:06.601463981Z","filename":"/pillar/zedcloud/send.go:206","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnAllIntf"}{"severity":"error","source":"zedagent","iid":"2151","content":"client.Do fail: ECONNREFUSED","msgid":"2675","timestamp":"2023-09-27T07:07:06.615360421Z","filename":"/pillar/zedcloud/send.go:974","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnIntf"}{"severity":"error","source":"zedagent","iid":"2151","content":"All attempts to connect to https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/info failed: send via eth0 with src IP 192.168.10.168: Post \"https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/info\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2676","timestamp":"2023-09-27T07:07:06.615438982Z","filename":"/pillar/zedcloud/send.go:1127","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnIntf"}{"severity":"error","source":"zedagent","iid":"2151","content":"All attempts to connect to 192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/info failed: send via eth0 with src IP 192.168.10.168: Post \"https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/info\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2677","timestamp":"2023-09-27T07:07:06.615489128Z","filename":"/pillar/zedcloud/send.go:266","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnAllIntf"}{"severity":"info","source":"zedagent","iid":"2151","content":"handleDeferred() the rest to be sent: 1","msgid":"2678","timestamp":"2023-09-27T07:07:06.615531352Z","filename":"/pillar/zedcloud/deferred.go:227","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.(*DeferredContext).handleDeferred"}{"severity":"info","source":"zedagent","iid":"2151","content":"deferred queue has INFO: 1","msgid":"2679","timestamp":"2023-09-27T07:07:06.615561917Z","filename":"/pillar/cmd/zedagent/zedagent.go:2472","function":"github.com/lf-edge/eve/pkg/pillar/cmd/zedagent.getDeferredSentHandlerFunction.func1"}{"severity":"info","source":"zedagent","iid":"2151","content":"processQueueTask: some deferred items remain to be sent","msgid":"2680","timestamp":"2023-09-27T07:07:06.615594359Z","filename":"/pillar/zedcloud/deferred.go:125","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.(*DeferredContext).processQueueTask"}{"severity":"warning","source":"zedagent","iid":"2151","content":"All management ports are marked failed; trying all","msgid":"2681","timestamp":"2023-09-27T07:07:09.528024684Z","filename":"/pillar/zedcloud/send.go:206","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnAllIntf"}{"severity":"error","source":"zedagent","iid":"2151","content":"client.Do fail: ECONNREFUSED","msgid":"2682","timestamp":"2023-09-27T07:07:09.562542053Z","filename":"/pillar/zedcloud/send.go:974","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnIntf"}{"severity":"error","source":"zedagent","iid":"2151","content":"All attempts to connect to https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/config failed: send via eth0 with src IP 192.168.10.168: Post \"https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/config\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2683","timestamp":"2023-09-27T07:07:09.562690421Z","filename":"/pillar/zedcloud/send.go:1127","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnIntf"}{"severity":"error","source":"zedagent","iid":"2151","content":"All attempts to connect to 192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/config failed: send via eth0 with src IP 192.168.10.168: Post \"https://192.168.10.129:9095/api/v2/edgedevice/id/848665bf-8424-481d-ac8c-a3d047ca0fdf/config\": dial tcp 192.168.10.168:0->192.168.10.129:9095: connect: connection refused","msgid":"2684","timestamp":"2023-09-27T07:07:09.562780451Z","filename":"/pillar/zedcloud/send.go:266","function":"github.com/lf-edge/eve/pkg/pillar/zedcloud.SendOnAllIntf"}{"severity":"info","source":"zedagent","iid":"2151","content":"config processing flag is not OK: configReqFailed","msgid":"2685","timestamp":"2023-09-27T07:07:10.332505737Z","filename":"/pillar/cmd/zedagent/handleconfig.go:460","function":"github.com/lf-edge/eve/pkg/pillar/cmd/zedagent.configTimerTask"}